analysis/topEFT/topeft.py

```diff
@@ -252,7 +252,7 @@ def process(self, events):
 
         # Build FO collection
         m_fo = mu[mu.isPres & mu.isLooseM & mu.isFO]
-        e_fo = e[e.isPres & e.isLooseE & e.isFO]
+        e_fo = ele[ele.isPres & ele.isLooseE & ele.isFO]
 
         # Attach the lepton SFs to the electron and muons collections
         AttachElectronSF(e_fo,year=year)
```

Comment:
Looks like there's still a few that are just `e`

Reply:
Fair enough. I'm fine with `e_` and `m_`, just wasn't sure if the point of the PR was to replace _all_ `e` with `ele` or just the actual object `e.`.